code/modules/clothing/suits/armor_vr.dm

```diff
@@ -33,6 +33,48 @@
 				H << "<span class='warning'>You need to have a wolf-taur half to wear this.</span>"
 				return 0
 
+	kate
```

Comment:
If I'm correct, this is just a fancy way to say that it's a child of "/obj/item/clothing/suit/armor/vest/wolftaur" without making it "/obj/item/clothing/suit/armor/vest/wolftaur/kate"